toolchain/lower/handle.cpp

```diff
@@ -60,6 +60,12 @@ auto HandleAssign(FunctionContext& context, SemIR::InstId /*inst_id*/,
   context.FinishInit(storage_type_id, inst.lhs_id, inst.rhs_id);
 }
 
+auto HandleAssociatedEntity(FunctionContext& context, SemIR::InstId inst_id,
+                            SemIR::AssociatedEntity /*inst*/) -> void {
+  context.SetLocal(inst_id,
+                   llvm::ConstantStruct::getAnon(context.llvm_context(), {}));
```

Comment:
Should there be lowering tests?

Reply:
Added, and in fact this didn't work :) Fixed.